lldb/source/API/SBTarget.cpp

```diff
@@ -1764,8 +1764,8 @@ lldb::SBSymbolContextList SBTarget::FindFunctions(const char *name,
   function_options.include_inlines = true;
 
   FunctionNameType mask = static_cast<FunctionNameType>(name_type_mask);
-  target_sp->GetImages().FindFunctions(ConstString(name), mask,
-                                       function_options, *sb_sc_list);
+  target_sp->GetImages().FindFunctions(
+      ConstString(name), mask, function_options, SymbolContext(), *sb_sc_list);
```

Comment:
If we use the default parameter as suggested above, all changes in this file go away.